oger/doc/txt.py

```diff
@@ -141,3 +142,17 @@ def iter_documents(self, source):
                     continue  # member is not a regular file
                 id_ = basename(member.name)
                 yield self._document(stream, id_)
+
+
+class TXTTSVLoader(DocIterator, _TXTLoaderMixin):
+    '''
+    Loader for single TSV document consisting of 
+    ColumnA -> [id]
+    ColumnB -> [text] format
+    '''
+    def iter_documents(self, source):
+        with open(source, 'r') as f:
```

Comment:
```suggestion
        with text_stream(source) as f:
```
This is a bit more flexible and also accepts an open file or a URL.